package.json

```diff
@@ -21,10 +21,10 @@
     "lint": "yarn eslint --max-warnings=0 && yarn prettier && yarn stylelint",
     "lint:fix": "yarn stylelint --fix && yarn eslint --max-warnings=0 --fix && yarn prettier --write",
     "clean:storybook": "rimraf 'storybook/public'",
-    "clean": "lerna run clean && yarn clean:storybook",
+    "clean": "yarn turbo clean && yarn clean:storybook",
```

Comment:
Using turbo to go through all directories and execute their internal clean functions, this is to support the different clean configs in different packages.